tuxemon/technique.py

```diff
@@ -76,17 +75,17 @@ class Technique:
     """A technique object is a particular skill that tuxemon monsters can use
     in battle.
     """
-
     def __init__(
         self,
         slug: Optional[str] = None,
         carrier: Optional[Monster] = None,
         link: Optional[Monster] = None,
     ) -> None:
         self._combat_counter = 0  # number of turns that this technique has been active
+        self._life_counter = 0
         self.accuracy = 0.0
         self.animation = ""
-        self.can_apply_status = False
+        self.can_apply_status = True
```

Comment:
Why was the default changed?

Reply:
it was causing problems with the original overfeed code, i changed it back as that code no longer exists.